ares/ares/node/object.hpp

```diff
@@ -1,14 +1,14 @@
-//identifier is static, allowing template<typename T> to access via T::identifier
+//identifier() is static, allowing template<typename T> to access via T::identifier()
 //identity() is virtual, allowing T* to access via T->identity()
 
 #define DeclareClass(Type, Name) \
-  static inline const string identifier = Name; \
+  static auto identifier() -> string { return Name; } \
```

Comment:
Does Clang not support constants in a class?

Reply:
It does, but there is an issue with static initialization order. Please see the associated commit message.